1228-public_page/user_navigates_to_landing_page_for_registration_and_can_see_files.feature

```diff
@@ -0,0 +1,51 @@
+Feature: User navigates to Landing Page for Registration and can see files
+
+    @1530
+    Scenario: Files that are Administrative Agreements are hidden
+        Given a User views the Landing Page for a Registration
```

Comment:
Den Given'en her finnes ikke som scenario... Enten burde det bli brukt en Given som vi har fra før, eller så må vi opprette et scenario for `User views the Landing Page for a Registration`.

Reply:
Det er vel denne:
https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/1228-public_page/881-view_public_page_for_publication.feature
Burde vært revidert slik at den bruker riktig terminologi

Reply:
Ja - kan du oppdatere den også, @elkebab?

Reply:
Men _må_ en Given peke til et annet Scenario? Hvordan skal da Given til det aller første scenarioet være? 🤯

Reply:
Det skal helst det ja. Første scenario bør være hvordan du navigerer til appen. Noe ala "Given User navigates to nva.unit.no" liksom